src/main/java/com/yahoo/bullet/record/BulletRecord.java

```diff
@@ -28,8 +28,8 @@
  * The record currently supports these fields:
  * <pre>
  * Primitives: Boolean, Integer, Long, Float, Double, String
- * Complex: {@code Map <String, "Primitives">, Map<String, Map<String, "Primitives">>, List<Map<String, "Primitives">},
- *           where "Primitives" refers to the afore-mentioned Primitives.
+ * Complex: {@code Map <String, "Primitives">, Map<String, Map<String, "Primitives">>, List<Map<String, "Primitives">,
+ *           List<"Primitives">} where "Primitives" refers to the afore-mentioned Primitives.
```

Comment:
Move this before the List<Map<String...>?

Reply:
ok